staging/src/k8s.io/library-go/README.md

```diff
@@ -0,0 +1,23 @@
+# library-go
+
+This repository provides helpers primarily for core components (core components as described in [Create a k8s.io/component-base repo](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cluster-lifecycle/wgs/0032-create-a-k8s-io-component-repo.md#component-definition)) which are required by at least two components living outside
+of the main `k8s.io/kubernetes` package. Yet, still with a high level of abstraction.
+
+The only allowed kubernetes dependencies are `k8s.io/apimachinery`, `k8s.io/api` and `k8s.io/client-go`.
```

Comment:
Is this being enforced?

Reply:
(nvm I see it is a TODO at the top)

Reply:
Updated `staging/publishing/import-restrictions.yaml` which should do this automatically.